OverReact_Redux_Documentation.md

```diff
@@ -0,0 +1,260 @@
+# OverReact Redux
+> A Dart React Redux wrapper for targeted state updates.
+
+---
+* __[Purpose](#purpose)__
+    * [Advantages](#advantages)
+* __[Using it in your project](#using-it-in-your-project)__
+* __[ReduxProvider](#reduxprovider)__
+* __[The Connect Function](#the-connect-function)__
+    * [Connect Parameters](#connect-parameters)
+        * [mapStateToProps](#mapstatetoprops)
+        * [mapDispatchToProps](#mapdispatchtoprops)
+        * [mergeProps](#mergeprops)
+        * [areStatesEqual](#arestatesequal)
+        * [areOwnPropsEqual](#areownpropsequal)
+        * [areStatePropsEqual](#arestatepropsequal)
+        * [areMergedPropsEqual](#aremergedpropsequal)
+        * [context](#context)
+* __[Using Multiple Stores](#using-multiple-stores)__
+* __[Using Redux DevTools](#using-redux-devtools)__
+
+## Purpose
+[React Redux](https://react-redux.js.org) is a UI binding library that allows for better Redux integration with React. 
+This integration allows for APIs that align more closely with React design patterns while also providing more direct 
+access to store data. OverReact Redux is the Dart wrapper around that library.
+
+### Advantages
+
+A full list of advantages can be found [here](https://react-redux.js.org/introduction/why-use-react-redux). 
+Especially useful for OverReact though, OverReact Redux allows for targeted updates that only update components 
+that receive data which has changed, rather than the whole component tree (as is the behavior with React). By utilizing 
+the `connect()` function in conjunction with `mapStateToProps()`, a component will only update when a piece of information 
+it uses is updated.
+
+
+## Using it in your project
```

Comment:
This document should have a big bold callout near the beginning that explains how to serve / play around with our functioning demo component.